noxfile.py

```diff
@@ -99,6 +108,8 @@ def format(session):
     # Ensure that pre-commit itself ran successfully
     assert process.returncode in (0, 1)
 
+    lint(session)
```

Comment:
Not related to this PR directly but I noticed this got removed after the pre-commit case, imo it makes sense to call `lint` after `format` to ensure your changes post-format are good to go.